main/src/main/res/drawable/ic_menu_edit_lists.xml

```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Didn't realize the icon does already exist (which also raises the question of why the icon has not yet been used in suitable places ;-))

Fixed.

Reply:
> Didn't realize the icon does already exist (which also raises the question of why the icon has not yet been used in suitable places ;-))

I introduced it back when I created the filters. It is in code repo since then.

 I also used it in my first version of the "select list" dialog as default icon to show when the list has no marker assigned. But this was not deemed approbiate back then, thus I changed it to the "list symbol" (w/o the pencil in it) which is still in use there